spec.html

```diff
@@ -17401,7 +17401,7 @@ <h2>Syntax</h2>
           `u` `{` CodePoint [lookahead &notin; HexDigit] [lookahead != `}`]
 
         NotCodePoint ::
-          HexDigits[~Sep] [> but only if MV of |HexDigits| > 0x10FFFF]
+          HexDigits[~Sep] [> but only if MV of |HexDigits| &gt; 0x10FFFF]
```

Comment:
this seems like something ecmarkup should be able to do - also there's another `>` earlier in the line, would that need it too?

Reply:
> this seems like something ecmarkup should be able to do

It seems to already be handled for output, but this corrects the existing _source_ inconsistency w.r.t. `[> but only if … &le; …]` as observable three lines down. Extension to linting and/or formatting would make sense.

> also there's another `>` earlier in the line, would that need it too?

No, because literal `[> …]` is [grammarkdown prose assertion syntax](https://rbuckton.github.io/grammarkdown/usage/syntax.html#assertions) (and is [converted to `<emu-gmod>…</emu-gmod>` on output](https://github.com/rbuckton/grammarkdown/blob/061b7b1c0e39fc9462add8a64e3920a3d5212921/src/emitter/ecmarkup.ts#L349)).

Reply:
Grammarkdown accepts `[&gt;`: https://github.com/rbuckton/grammarkdown/issues/60

But it also accepts `>`, in both places. I kind of prefer it to be `>` but I recognize we're inconsistent.

Reply:
Should I revert changes to this line then? emu-format isn't happy with the `&gt;` anyway.

Reply:
I just did it. The main goal here is to indicate use of `-` rather than `–` or `−`.